docs/static/releasenotes.asciidoc

```diff
@@ -3,12 +3,18 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-5-6-5,Logstash 5.6.5>>
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-5]]
+=== Logstash 5.6.5 Release Notes
+* Fix bug where persistent queue could not be opened when in some states
+* Ensure logstash terminates on second sigint, there were some situations where executing two sigints would not kill Logstash
```

Comment:
Change to: Ensure that Logstash terminates on second SIGINT. Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash. (you might want to format SIGINT with back ticks...I can't remember whether we do elsewhere).

Reply:
Improved shutdown process by ensuring that.... (related to my comment below about making these items past tense)